src/Hl7.Fhir.Base/Model/PocoNodeExtensions.cs

```diff
@@ -335,4 +346,20 @@ internal static string SerializeToString(this PocoNode pn, bool pretty)
 
         return serializer.SerializeToString(pn.Poco, pretty, rootName: rootName);
     }
+
+    /// <summary>
+    /// Converts the specified PocoNode instance to its JSON representation.
+    /// </summary>
+    /// <param name="pn">The PocoNode instance to serialize.</param>
+    /// <param name="pretty">Indicates whether the JSON output should be formatted for readability.</param>
+    /// <returns>A JSON string representing the given PocoNode.</returns>
+    public static string ToJson(this PocoNode pn, bool pretty)
```

Comment:
This functionality was already accessible through the ITypedElement interface: `Hl7.Fhir.ElementModel.TypedElementExtensions.ToJson` (the other change in the PR). It was not added on PocoNode before. Was that simply an oversight, or for some reason deliberate?